src/Orleans.Core.Abstractions/Placement/ActivationCountBasedPlacement.cs

```diff
@@ -3,7 +3,7 @@
 namespace Orleans.Runtime
 {
     [Serializable]
-    internal class ActivationCountBasedPlacement : PlacementStrategy
```

Comment:
What's the reason for making all these built-in strategies public? Seems like it's not something that application code will need to reference.